Source/Fluxor/DependencyInjection/FluxorOptions.cs

```diff
@@ -14,6 +14,17 @@ public class FluxorOptions
 		internal AssemblyScanSettings[] AssembliesToScan { get; private set; } = Array.Empty<AssemblyScanSettings>();
 		internal Type[] TypesToScan { get; private set; } = Array.Empty<Type>();
 		internal Type[] MiddlewareTypes = Array.Empty<Type>();
+		/// <summary>
+		/// The Lifecycle that should be used when registering Fluxor features/reducers/effects/middleware</br>
+		/// (default) LifecycleEnum.Scoped = Create a new instance for each new request</br>
+		/// LifecycleEnum.Singleton = Create a new instance on first request and reuse for rest of application lifetime</br>
```

Comment:
I think comments are better without `</br>`

Reply:
removed the br's